tests/buildcff2vf_test.py

```diff
@@ -113,3 +115,83 @@ def test_bug1003_compat():
     actual_ttx = generate_ttx_dump(actual_path, ['CFF2'])
     expected_ttx = get_expected_path('bug1003.ttx')
     assert differ([expected_ttx, actual_ttx, '-s', '<ttFont sfntVersion'])
+
+
+def test_stat_axis_not_in_fvar():
+    """
+    Fail if fvar axes are missing from STAT
+    """
+    input_dir = get_input_path('STAT_tests')
+    temp_dir = get_temp_dir_path('STAT_tests')
+    copytree(input_dir, temp_dir)
+    ds_path = os.path.join(temp_dir,
+                           'STAT_axis_missing/STAT_axis_missing.designspace')
+    try:
+        runner(CMD + ['-o', 'd', f'_{ds_path}'])
+    except subprocess.CalledProcessError:
+        pytest.xfail()
```

Comment:
Let's use:
```python
with pytest.raises(subprocess.CalledProcessError):
    runner(CMD + ['-o', 'd', f'_{ds_path}'])
```
With try/except, the `runner` could succeed and this test would still pass which is probably not desired.

Reply:
Yep, little better now.